testutil/sample/launch.go

```diff
@@ -87,6 +99,45 @@ func Request(chainID string) *launch.Request {
 	}
 }
 
+// AllRequestContents creates all contents types for request and
+// returns a list of all pack contents converted to `types.Any` object
+func AllRequestContents(chainID, genesis, vested, validator string) []*types.Any {
+	contents := make([]proto.Message, 0)
+	contents = append(contents, GenesisAccount(chainID, genesis))
```

Comment:
Can't we use a single append for all?

```
append(contents,
GenesisAccount(chainID, genesis),
AccountRemoval(genesis),
...)
```

Aren't we gonna drop Any yet?

Reply:
> Aren't we gonna drop Any yet?

No, we're waiting Frojdi coming back to have his mind about this.
For now we finish the messages implementation using Any

Reply:
@ilgooz, done on [`5906f90`](https://github.com/tendermint/spn/pull/235/commits/5906f90b4fc8f4214db3683ec2cd7fcdb15441d6)